### ISSUE TITLE
Safety Check Reports 2 Vulnerabilities: CVE-2020-7694 and CVE-2020-7695

### ISSUE THREAD

**Issue by societymartingale** [Kludex/uvicorn#749]
### Describe the bug

Safety check reports the following 2 vulnerabilities:

- https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-7694
- https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-7695

### To reproduce
```
tmp> python3.8 -m venv uvi
tmp> source uvi/bin/activate
(uvi) tmp> pip install uvicorn
(uvi) tmp> pip install safety
(uvi) tmp> pip list
Package    Version
---------- ---------
certifi    2020.6.20
chardet    3.0.4
click      7.1.2
dparse     0.5.1
h11        0.9.0
httptools  0.1.1
idna       2.10
packaging  20.4
pip        19.2.3
pyparsing  2.4.7
PyYAML     5.3.1
requests   2.24.0
safety     1.9.0
setuptools 41.2.0
six        1.15.0
toml       0.10.1
urllib3    1.25.10
uvicorn    0.11.8
uvloop     0.14.0
websockets 8.1
(uvi) tmp> safety check --full-report
+==============================================================================+
| REPORT                                                                       |
| checked 20 packages, using default DB                                        |
+============================+===========+==========================+==========+
| package                    | installed | affected                 | ID       |
+============================+===========+==========================+==========+
| uvicorn                    | 0.11.8    | >=0                      | 38624    |
+==============================================================================+
| This affects all versions of package uvicorn. The request logger provided by |
| the package is vulnerable to ASNI escape sequence injection. Whenever any    |
| HTTP request is received, the default behaviour of uvicorn is to log its     |
| details to either the console or a log file. When attackers request crafted  |
| URLs with percent-encoded escape sequences, the logging component will log   |
| the URL after it's been processed with urllib.parse.unquote, therefore       |
| converting any percent-encoded characters into their single-character        |
| equivalent, which can have special meaning in terminal emulators. By         |
| requesting URLs with crafted paths, attackers can: * Pollute uvicorn's       |
| access logs, therefore jeopardising the integrity of such files. * Use ANSI  |
| sequence codes to attempt to interact with the terminal emulator that's      |
| displaying the logs (either in real time or from a file). See:               |
| CVE-2020-7694.                                                               |
+==============================================================================+
| uvicorn                    | 0.11.8    | >=0                      | 38625    |
+==============================================================================+
| This affects all versions of package uvicorn. Uvicorn's implementation of    |
| the HTTP protocol for the httptools parser is vulnerable to HTTP response    |
| splitting. CRLF sequences are not escaped in the value of HTTP headers.      |
| Attackers can exploit this to add arbitrary headers to HTTP responses, or    |
| even return an arbitrary response body, whenever crafted input is used to    |
| construct HTTP headers. See: CVE-2020-7695.                                  |
+==============================================================================+
```

### Expected behavior

Expect no issues to be found

### Actual behavior

2 issues found

### Environment

(uvi) tmp> uvicorn --version
Running uvicorn 0.11.8 with CPython 3.8.2 on Darwin

**Comment by euri10** [comment]
This is fixed in 0.11.7.

Please take the time to read and fill entirely the template next time.

**Comment by societymartingale** [comment]
OK, the issue is actually on the Safety tool's side. Their free database is updated only once per month. https://github.com/pyupio/safety-db/issues/2314 . Sorry for the false alarm.
